using_packages/conanfile_txt.rst

```diff
@@ -137,6 +137,14 @@ and this is the linkage that should be used if consumers don't specify otherwise
 
         $ conan get Poco/1.9.0@pocoproject/stable
 
+    That outputs the whole package recipe, where you can see the options and how they are used for the build.
```

Comment:
Please rephrase to something more direct:

"To see only specific fields of the recipe you can use the :command:`conan inspect` command instead:"